integration-tests/rest/src/test/java/org/apache/camel/quarkus/component/rest/it/RestTest.java

```diff
@@ -25,9 +25,11 @@
 import static org.hamcrest.CoreMatchers.equalTo;
 import static org.hamcrest.Matchers.is;
 import static org.hamcrest.Matchers.matchesPattern;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
 @QuarkusTest
 class RestTest {
+    private static final Person person = new Person("John", "Doe", 64);
```

Comment:
Is it a requirement to use the same object for json/xml consumer/producer binding ? Otherwise, I would use distinct objects, and maybe with different value.

Reply:
it's definitely not a requirement. I just thought it would be a little better to create the object only once. I'll change it as you suggest.